tools/syncer.md

```diff
@@ -58,49 +58,63 @@ binlog-gtid = "2bfabd22-fff7-11e6-97f7-f02fa73bcb01:1-23,61ccbb5d-c82d-11e6-ac2e
 Description of Syncer command line options:
 
 ```
-Usage of Syncer:
+Usage of syncer:
   -L string
-      log level: debug, info, warn, error, fatal (default "info")
-  -V
-      to print Syncer version info (default false)
-  -auto-fix-gtid
-      to automatically fix the gtid info when MySQL master and slave switches (default false)
+        log level: debug, info, warn, error, fatal (default "info")
+  -V    to print Syncer version info (default false)
   -b int
-      the size of batch transactions (default 10)
+        the size of batch transactions (default 10)
   -c int
-      the number of batch threads that Syncer processes (default 16)
+        the number of batch threads that Syncer processes (default 16)
   -config string
-      to specify the corresponding configuration file when starting Syncer; for example, `--config config.toml`
+        path to config file
+  -enable-ansi-quotes
+        to specify the corresponding configuration file when starting Syncer; for example, `--config config.toml`
   -enable-gtid
-      to start Syncer using the mode; default false; before enabling this option, you need to enable GTID in the upstream MySQL
+        to start Syncer using the mode; default false; before enabling this option, you need to enable GTID in the upstream MySQL
+  -flavor string
+        use flavor for different MySQL source versions; support "mysql", "mariadb" now; if you replicate from mariadb, please set it to "mariadb" (default "mysql")
   -log-file string
-      to specify the log file directory, such as `--log-file ./syncer.log`
+        to specify the log file directory, such as `--log-file ./syncer.log`
   -log-rotate string
-      to specify the log file rotating cycle, hour/day (default "day")
+        to specify the log file rotating cycle, hour/day (default "day")
+  -max-retry int
+        to specify maxinum retry when network interruption (default 100)
   -meta string
-      to specify the meta file of Syncer upstream (in the same directory with the configuration file by default "syncer.meta")
+        to specify the meta file of Syncer upstream (in the same directory with the configuration file by default "syncer.meta")
+  -persistent-dir string
+        to specify  syncer history table structures persistent dir; set to non-empty string will choosing history table structure according to column length when constructing DML
```

Comment:
in fact, it's *file* path, not *dir*.

Reply:
Users are bound to keep wondering and asking